src/components/commons/inputs/FormDropdown.tsx

```diff
@@ -22,12 +22,25 @@ type FormDropdownProps = {
   label?: string;
   isSmall?: boolean;
   isDisabled?: boolean;
+  color?: 'purple' | 'yellow';
 };
 
-export const FormDropdown = ({ name, label = '', options, isSmall = false, isDisabled = false }: FormDropdownProps) => {
+export const FormDropdown = ({
+  name,
+  label = '',
+  options,
+  isSmall = false,
+  isDisabled = false,
+  color = 'purple',
+}: FormDropdownProps) => {
   const { register, setValue } = useFormContext();
   const { isOpen, popupRef, buttonRef, togglePopup } = useTogglePopup();
-  const [currentOptionTitle, setCurrentOptionTitle] = useState(options[0].title);
+  const [currentOptionTitle, setCurrentOptionTitle] = useState(options[0]?.title);
+
+  useEffect(() => {
+    setCurrentOptionTitle(options[0]?.title);
+    setValue(name, options[0]?.value, { shouldValidate: true });
+  }, [name, options, setValue]);
```

Comment:
```suggestion
  }, [name, options]);
```
setValue는 변할 일이 없어서 빼줘도 될 거 같습니다

Reply:
네! 반영했습니다~